pkg/subctl/cmd/deploybroker.go

```diff
@@ -38,6 +38,10 @@ import (
 	"github.com/submariner-io/submariner-operator/pkg/subctl/operator/submarinerop"
 )
 
+const (
+	DefaultBrokerNamespace = "submariner-k8s-broker"
```

Comment:
This doesn't need to be exported so let's not leak it.

```suggestion
	defaultBrokerNamespace = "submariner-k8s-broker"
```